src/Squisher.js

```diff
@@ -52,16 +52,29 @@ class Squisher {
     }
 
     updatePixelBoard() {
-        this.pixelBoard = new Array(8 * this.entities.length);
+        const entityFrameSize = 46;
+        this.pixelBoard = new Array(entityFrameSize * this.entities.length);
         for (let i = 0; i < this.entities.length; i++) {
-            this.pixelBoard[8 * i] = this.entities[i].color[0];
-            this.pixelBoard[8 * i + 1] = this.entities[i].color[1];
-            this.pixelBoard[8 * i + 2] = this.entities[i].color[2];
-            this.pixelBoard[8 * i + 3] = this.entities[i].color[3];
-            this.pixelBoard[8 * i + 4] = this.entities[i].pos.x * 100;
-            this.pixelBoard[8 * i + 5] = this.entities[i].pos.y * 100;
-            this.pixelBoard[8 * i + 6] = this.entities[i].size.x * 100;
-            this.pixelBoard[8 * i + 7] = this.entities[i].size.y * 100;
+            this.pixelBoard[entityFrameSize * i] = this.entities[i].color[0];
+            this.pixelBoard[entityFrameSize * i + 1] = this.entities[i].color[1];
+            this.pixelBoard[entityFrameSize * i + 2] = this.entities[i].color[2];
+            this.pixelBoard[entityFrameSize * i + 3] = this.entities[i].color[3];
+            this.pixelBoard[entityFrameSize * i + 4] = Math.floor(this.entities[i].pos.x * 100);
+            this.pixelBoard[entityFrameSize * i + 5] = this.entities[i].pos.x * 10000 % 100;
+            this.pixelBoard[entityFrameSize * i + 6] = Math.floor(this.entities[i].pos.y * 100);
+            this.pixelBoard[entityFrameSize * i + 7] = this.entities[i].pos.y * 10000 % 100;
+            this.pixelBoard[entityFrameSize * i + 8] = Math.floor(this.entities[i].size.x * 100);
+            this.pixelBoard[entityFrameSize * i + 9] = this.entities[i].size.x * 10000 % 100;
+            this.pixelBoard[entityFrameSize * i + 10] = Math.floor(this.entities[i].size.y * 100);
+            this.pixelBoard[entityFrameSize * i + 11] = this.entities[i].size.y * 10000 % 100;
```

Comment:
I'm not 1 hundo on this, 
looking at: 
https://www.khanacademy.org/computing/computer-science/cryptography/modarithmetic/a/modular-multiplication 
we get:
A * 10000 % 100 =  (A % 100 * 10000 % 100) % 100 =  (A % 100 * 0) % 100 = 0
Where A is any number